.github/workflows/velox.yml

```diff
@@ -0,0 +1,57 @@
+name: Velox Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  vm:
+    runs-on: ubuntu-latest
+
+  container:
+    runs-on: ubuntu-latest
+    container: prestocpp/velox-torcharrow:kpai-20220228
```

Comment:
What is the container here?

Reply:
Its public velox image : https://hub.docker.com/r/prestocpp/velox-torcharrow/tags